src/plugins/data/public/ui/search_bar/search_bar.test.tsx

```diff
@@ -116,41 +118,46 @@ function wrapSearchBarInContext(testProps: any) {
 }
 
 describe('SearchBar', () => {
+  const SEARCH_BAR_TEST_ID = 'globalQueryBar';
   const SEARCH_BAR_ROOT = '.globalQueryBar';
-  const FILTER_BAR = '.filterBar';
+  const FILTER_BAR = '.globalFilterBar';
```

Comment:
TBH I have no clue how this test worked previously with the `.filterBar` class... I couldn't find it anywhere in the DOM. So I replaced it with `.globalFilterBar` which definitely exists 🙂